### ISSUE TITLE
Broken in latest update (Beta 2024.6.10.1405)

### ISSUE THREAD

**Issue by Nytra** [hantabaru1014/BetterInventoryBrowser#9]
Inventory does not work

[DESKTOP-H976HO2 - 2024.6.10.1405 - 2024-06-11 01_23_32.log](https://github.com/user-attachments/files/15780457/DESKTOP-H976HO2.-.2024.6.10.1405.-.2024-06-11.01_23_32.log)

```
Exception running OnAttach behavior for Component: Element: ID6693400, Type: FrooxEngine.InventoryBrowser, World: Userspace, IsRemoved: False, IsDestroyed: False, IsDisposed: False, Enabled: True
Element: ID6692000, Type: FrooxEngine.Slot, World: Userspace, IsRemoved: False, Slot name: Content, T: [0; 0; 0], R: [0; 0; 0; 1], S: [1; 1; 1], ActiveSelf: True, IsDestroyed: False
Element: ID6690100, Type: FrooxEngine.Slot, World: Userspace, IsRemoved: False, Slot name: TiledRawImage, T: [0; 0; 0], R: [0; 0; 0; 1], S: [1; 1; 1], ActiveSelf: True, IsDestroyed: False
Element: ID43BAF00, Type: FrooxEngine.Slot, World: Userspace, IsRemoved: False, Slot name: Canvas, T: [0; 0; 0], R: [0; 0; 0; 1], S: [1; 1; 1], ActiveSelf: True, IsDestroyed: False
Element: ID43BA400, Type: FrooxEngine.Slot, World: Userspace, IsRemoved: False, Slot name: Screen, T: [0; 0; 0], R: [0; 0; 0; 1], S: [0.000869414; 0.000869414; 0.000869414], ActiveSelf: True, IsDestroyed: False
Element: ID42B1400, Type: FrooxEngine.Slot, World: Userspace, IsRemoved: False, Slot name: Inventory, T: [0; 0; 0], R: [0; 0; 0; 1], S: [1; 1; 1], ActiveSelf: True, IsDestroyed: False
Element: ID292000, Type: FrooxEngine.Slot, World: Userspace, IsRemoved: False, Slot name: Screens, T: [0; 0.01877934; 0], R: [0; 0; 0; 1], S: [1; 1; 1], ActiveSelf: True, IsDestroyed: False
Element: ID291500, Type: FrooxEngine.Slot, World: Userspace, IsRemoved: False, Slot name: ScreensRoot, T: [0; 0; 0], R: [0; 0; 0; 1], S: [1; 1; 1], ActiveSelf: True, IsDestroyed: False
Element: ID287900, Type: FrooxEngine.Slot, World: Userspace, IsRemoved: False, Slot name: Render, T: [0; 0; 0], R: [0; 0; 0; 1], S: [1; 1; 1], ActiveSelf: True, IsDestroyed: False
Element: ID283200, Type: FrooxEngine.Slot, World: Userspace, IsRemoved: False, Slot name: Dash, T: [0; 0; 0], R: [1.520568E-16; 1; 0; -4.371139E-08], S: [1; 1; 1], ActiveSelf: True, IsDestroyed: False
Element: ID282700, Type: FrooxEngine.Slot, World: Userspace, IsRemoved: False, Slot name: DashRotation, T: [0; 0; 0], R: [0; 0.9890949; -0.14728; 0], S: [1; 1; 1], ActiveSelf: True, IsDestroyed: False
Element: ID281C00, Type: FrooxEngine.Slot, World: Userspace, IsRemoved: False, Slot name: DashOffset, T: [0; 0.4181766; 1.373058], R: [0; 0; 0; 1], S: [1.727869; 1.727859; 1.727869], ActiveSelf: True, IsDestroyed: False
Element: ID27DD00, Type: FrooxEngine.Slot, World: Userspace, IsRemoved: False, Slot name: Dash, T: [0; 1.625; 0.1], R: [0; 0; 0; 1], S: [1; 1; 1], ActiveSelf: True, IsDestroyed: False
Element: ID2400, Type: FrooxEngine.Slot, World: Userspace, IsRemoved: False, Slot name: Root, T: [0; 0; 0], R: [0; 0; 0; 1], S: [1; 1; 1], ActiveSelf: True, IsDestroyed: False
Element: ID0, Type: FrooxEngine.World, World: Userspace, IsRemoved: False

System.TypeLoadException: Could not resolve type with token 01000063 (from typeref, class/assembly SkyFrost.Base.User, SkyFrost.Base, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null)
  at BetterInventoryBrowser.BetterInventoryBrowser+BrowserDialog_Patch.OnAttach_Postfix (FrooxEngine.BrowserDialog __instance, FrooxEngine.SyncRef`1[T] ____swapper, FrooxEngine.SyncRef`1[T] ____buttonsRoot) [0x001a9] in <bf68b8489330489fab952fa597153176>:0 
  at (wrapper dynamic-method) MonoMod.Utils.DynamicMethodDefinition.FrooxEngine.BrowserDialog.OnAttach_Patch1(FrooxEngine.BrowserDialog)
  at FrooxEngine.InventoryBrowser.OnAttach () [0x00000] in <44a8db68419d427182b7e1f08b8188e3>:0 
  at FrooxEngine.ComponentBase`1[C].RunOnAttach () [0x0001a] in <44a8db68419d427182b7e1f08b8188e3>:0 

  at System.Environment.get_StackTrace () [0x00000] in <9577ac7a62ef43179789031239ba8798>:0 
  at Elements.Core.UniLog.Error (System.String message, System.Boolean stackTrace) [0x00000] in <bca837a3221f4f83ba8c0f0b3f278de5>:0 
  at FrooxEngine.DebugManager.Error (System.Object obj, System.Boolean stackTrace) [0x00000] in <44a8db68419d427182b7e1f08b8188e3>:0 
  at FrooxEngine.ComponentBase`1[C].RunOnAttach () [0x00000] in <44a8db68419d427182b7e1f08b8188e3>:0 
  at FrooxEngine.ContainerWorker`1[C].AttachComponentInternal (C component, System.Boolean runOnAttachBehavior, System.Action`1[T] beforeAttach) [0x00000] in <44a8db68419d427182b7e1f08b8188e3>:0 
  at FrooxEngine.ContainerWorker`1[C].AttachComponent[T] (System.Boolean runOnAttachBehavior, System.Action`1[T] beforeAttach) [0x00000] in <44a8db68419d427182b7e1f08b8188e3>:0 
  at FrooxEngine.LegacyRadiantScreenWrapper`1[T].OnShow () [0x00000] in <44a8db68419d427182b7e1f08b8188e3>:0 
  at FrooxEngine.RadiantDashScreen.Show () [0x00000] in <44a8db68419d427182b7e1f08b8188e3>:0 
  at FrooxEngine.RadiantDash.OnCommonUpdate () [0x00000] in <44a8db68419d427182b7e1f08b8188e3>:0 
  at FrooxEngine.ComponentBase`1[C].InternalRunUpdate () [0x00000] in <44a8db68419d427182b7e1f08b8188e3>:0 
  at FrooxEngine.UpdateManager.RunUpdates () [0x00000] in <44a8db68419d427182b7e1f08b8188e3>:0 
  at FrooxEngine.World.RefreshStep () [0x00000] in <44a8db68419d427182b7e1f08b8188e3>:0 
  at FrooxEngine.World.Refresh () [0x00000] in <44a8db68419d427182b7e1f08b8188e3>:0 
  at FrooxEngine.WorldManager.UpdateStep () [0x00000] in <44a8db68419d427182b7e1f08b8188e3>:0 
  at FrooxEngine.WorldManager.RunUpdateLoop () [0x00000] in <44a8db68419d427182b7e1f08b8188e3>:0 
  at FrooxEngine.Engine.UpdateStep () [0x00000] in <44a8db68419d427182b7e1f08b8188e3>:0 
  at FrooxEngine.Engine.RunUpdateLoop () [0x00000] in <44a8db68419d427182b7e1f08b8188e3>:0 
  at UnityFrooxEngineRunner.FrooxEngineRunner.UpdateFrooxEngine () [0x00000] in <9d8f158765ab4db18c978ae979158151>:0 
  at UnityFrooxEngineRunner.FrooxEngineRunner.Update () [0x00000] in <9d8f158765ab4db18c978ae979158151>:0 
```

**Comment by SlyTheFloof** [comment]
[SLYS-DESKTOP-PC - 2024.6.11.74 - 2024-06-10 23_35_40.log](https://github.com/user-attachments/files/15781887/SLYS-DESKTOP-PC.-.2024.6.11.74.-.2024-06-10.23_35_40.log)

**Comment by Krzeszny** [comment]
Broken indeed.

**Comment by hantabaru1014** [comment]
Thanks your reports!  
Fixed in v0.6.3
https://github.com/hantabaru1014/BetterInventoryBrowser/releases/tag/v0.6.3
